### ISSUE TITLE
[Feature]: "run from here"

### ISSUE THREAD

**Issue by andris1** [Ironclad/rivet#343]
### Feature Request

When adding to a bigger graph, appending/iterating on new functionality can mean waiting a good while to test a minor change. 

I know there's caching, but if you have RNGs in your path, caching doesn't work.

P.S. _Love_ the tool and the work on it. I've almost completely replaced key notebooks with Rivet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by abrenneke** [comment]
Agree! This has come up a few times and I've been thinking about how to approach it.

**Comment by remixer-dec** [comment]
+1 on this one, sometimes re-running the whole graph when you need to re-run a single node is very time-consuming
